website_and_docs/content/documentation/test_practices/testing_types.zh-cn.md

```diff
@@ -36,7 +36,24 @@ aliases: [
 对于Web应用程序, 
 可以通过模拟预期的结果, 直接使用Selenium来完成此测试的自动化. 
 可以通过记录/回放或通过本文档中说明的各种支持的语言来完成此模拟.
+### 集成测试
 
+集成测试验证系统的不同组件或模块之间的交互。 几个模块一起测试。 集成测试的目的是确保所有模块按预期集成并协同工作。 自动化集成测试有助于确保这些交互按预期工作并且集成组件能够正常运行。
+>例如，**_测试在电子商务网站上下订单并付款的流程。_**
+### Jíchéng cèshì
+
+jíchéng cèshì yànzhèng xìtǒng de bùtóng zǔjiàn huò mókuài zhī jiān de jiāohù. Jǐ gè mókuài yīqǐ cèshì. Jíchéng cèshì de mùdì shì quèbǎo suǒyǒu mókuài àn yùqí jíchéng bìng xiétóng gōngzuò. Zìdònghuà jíchéng cèshì yǒu zhù yú quèbǎo zhèxiē jiāohù àn yùqí gōngzuò bìngqiě jíchéng zǔjiàn nénggòu zhèngcháng yùnxíng.
```

Comment:
Hi @SripriyaPKulkarni,

You can set the text in English, translations will be done later for all other pages.

Can you replace this text with English, also I don't recognize this language as zh-cn